dagger/chainguard.go

```diff
@@ -307,7 +307,7 @@ func publishChainguardImage(
 	env := "dev"
 	if strings.Contains(imagePath, "registry.staging.replicated.com") {
 		env = "stage"
-	} else if strings.Contains(imagePath, "registry.replicated.com") {
+	} else if strings.Contains(imagePath, "registry.replicated.com") || strings.Contains(imagePath, "proxy.replicated.com") {
```

Comment:
I think this will actually never be the proxy? Because i think its always going to want to operate on the published image - not the proxied version. its gonna be the 3 images in publish.go ala - https://github.com/replicatedhq/replicated-sdk/blob/b33008f0bae4cce5c8eef660fc553156dc2c8187/dagger/publish.go#L109